app/helpers/application_helper.rb

```diff
@@ -65,4 +65,8 @@ def default_model_thumbnail(model)
     end
   end
 
+  def staff_url(doc_or_obj)
+    "https://ddr.lib.duke.edu/id/#{doc_or_obj.permanent_id}"
```

Comment:
I would probably make the hostname for the staff URL configurable (e.g., in local_env.yml) so it can be set correctly for the server environment -- e.g., ddr-dev.lib.duke.edu .